### PR TITLE
k8s topology probe

### PR DESCRIPTION
**Pull request by hunchback**

### CODE REVIEW

File: documentation/index.md [hunk 1]
```diff
@@ -27,15 +27,43 @@ Topology probes currently implemented :
 * LLDP
 * LibVirt
 
-## Topology connectors
+## Analayzer Topology Probes
```
Comment (lebauce) [inline documentation/index.md]: s/Analayzer/Analyzer/

File: documentation/index.md [hunk 2]
```diff
@@ -27,15 +27,43 @@ Topology probes currently implemented :
...
+
+Following are analyzer graph topology probes:
 
 * Neutron
 * Docker
-* Kubernetes
+* K8s (Kubernetes)
```
Comment (lebauce) [inline documentation/index.md]: Why not simply "Kubernetes" ?

Comment (lebauce) [reply documentation/index.md]: Maybe we could display the "internal" name for each probe, like :
* Neutron (`neutron`)
* Docker (`docker`)
* Kubernetes (`k8s`)
* Lxd (`lxd`)
* Opencontrail (`opencontrail`)
* runC (`runc`)

Comment (lebauce) [reply documentation/index.md]: By the way, Neutron, Docker Lxd, Opencontrail and runC are not analyzer probes, but agent probes.

File: documentation/index.md (hunk 1 above)
Comment (hunchback) [reply documentation/index.md]: ok

File: documentation/index.md (hunk 2 above)
Comment (hunchback) [reply documentation/index.md]: ok - will handle all the above

**Pull request merged**